README.md

```diff
@@ -10,9 +10,45 @@ Istio is an open-source service mesh that provides a uniform way to manage, conn
 
 The Istio module allows you to add Istio Operator to the Kyma runtime. Within Istio Operator, Istio Controller is responsible for installing, uninstalling, and managing Istio. For more information, read the [Istio Controller documentation](./docs/user/00-10-overview-istio-controller.md).
 
+## Install Istio Manager and Istio from the latest release
+
+### Prerequisites
+
+- Access to a Kubernetes (v1.24 or higher) cluster
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+### Procedure
+
+1. Clone the project and open the `istio` folder.
+
+   ```bash
+   git clone https://github.com/kyma-project/istio.git && cd istio/
+   ```
+
+2. To install Istio, you must install latest Istio Manager first. Run the following:
+
+   ```bash
+   kubectl create ns kyma-system
```

Comment:
As far as I know the namespace is included in the `istio-manager.yaml` asset.

Reply:
Yes, you are correct: https://github.com/kyma-project/istio/blob/18ae0c476fd96310cefc542cb6b4b043a459d50e/config/default/kustomization.yaml#L1-L2